convos/settings_gui.py

```diff
@@ -190,7 +187,7 @@ def prob_updater(update, context):  # PROBABILITY
     return PROBABILITY
 
 
-def change_prob(update, context):  # UPDATED
+def change_prob(update: Update, _: CallbackContext) -> int:  # UPDATED
```

Comment:
> What is the _ for?

Since `context` is not being used, setting it as _ saves some memory (very little).


> Is CallbackContext passed automatically to these functions?

Yep